### PR TITLE
Refactoring for CORSIKA and beam campaigns

### PR DESCRIPTION
**Pull request by soleti**
Small refactoring and moving concatenation after resampling for CORSIKA and CRY

### CODE REVIEW

File: CampaignConfig/mdc2020_corsika.cfg [hunk 1]
```diff
@@ -50,69 +51,69 @@ job_setup.postscript_9 = loggedMu2e.sh -c `cat torun.txt`
...
 executable.name = true
 executable_2.name = generate_fcl
-executable_2.arg_4 = --merge=50
-executable_2.arg_5 = --inputs=inputs.txt
-executable_2.arg_6 = --embed
-executable_2.arg_7 = template.fcl
-
+executable_2.arg_4 = --aux=1:physics.filters.cosmicResample.fileNames:inputs.txt
+executable_2.arg_5 = --run-number=1205
+executable_2.arg_6 = --events-per-job=200000
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_corsika.cfg:63]: This is OK for now, but in future there should be a script to compute it from the sam data

File: CampaignConfig/mdc2020_cry.cfg [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+[stage_resampler_cat_fcl]
+global.stage_name = CosmicCRYCat
+job_output.dest = %(outdir_fcl)s/%(stage_name)s/%(desc)s/fcl/
+
+job_setup.prescript = samweb list-file-locations --defname=dts.%(submitter)s.CosmicCRY.%(desc)s.art --schema=root | cut -f1 > inputs.txt
+job_setup.prescript_2 = echo '#include "Production/JobConfig/common/artcat.fcl"' >> template.fcl
+job_setup.prescript_3 = echo 'outputs.out.fileName: "dts.DSOWNER.CosmicCRYCat.DSCONF.SEQ.art"' >> template.fcl
+executable.name = true
+executable_2.name = generate_fcl
+executable_2.arg_4 = --merge=50
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_cry.cfg:87]: Don't you want to remove this as well?

Comment (soleti) [reply CampaignConfig/mdc2020_cry.cfg:87]: This is after the resampling

Comment (brownd1978) [reply CampaignConfig/mdc2020_cry.cfg:87]: OK thanks for clarifying.

**Review by brownd1978 — APPROVED** [review]

**Pull request merged**